thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2519,44 +2566,48 @@ void CMasterGraph::getFinalProgress()
         if (0 == msg.remaining())
             continue;
 
-        handleSlaveDone(sender-1, msg);
-
-        if (!queryOwner())
+        unsigned slavesPerProcess = globals->getPropInt("@slavesPerProcess", 1); // JCSMORE - should move somewhere common
```

Comment:
I think this is testing the wrong variable.   Should it be some combination of slavesPerNode and processPerSlave?